MVP/sample_job.yaml

```diff
@@ -0,0 +1,29 @@
+- job-template:
+    name: '{name}-{component}'
+    defaults: defaults-build
+    #node: 'jslave-of-your-own'
+    properties:
+        - ownership:
+            owner: lilu
+            co-owners:
+                - lilu
+                - lilu
```

Comment:
For MVP I don't think they would care. Its only the MVP participants who would be interested.